### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Upgrade to 21.12.3
- (partially) fix change_url script (we still have problems on multiple URL changes)
- Now the app preseves images when updating
- Update to PHP8

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -68,7 +68,7 @@ then
   	cp -a "$final_path/.env" "$tmpdir/.env"
 
 	# Remove the app directory securely
-	ynh_secure_remove --file="$final_path"
+	#ynh_secure_remove --file="$final_path"
 
 	# Download, check integrity, uncompress and patch the source from app.src
 	ynh_setup_source --dest_dir="$final_path"
```
Comment (alexAubin) [inline scripts/upgrade:74]: (Ideally, small improvement could be to use `--keep` for the .env file)

**Review by alexAubin — APPROVED** [review]

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by alexAubin** [comment]
Bumpity bump, shall we merge ? :eyes:

**Comment by ericgaspar** [comment]
trying to solve upgrade from revious versions...

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!tastemetitus

**Comment by ericgaspar** [comment]
!tastemetitus

**Pull request merged**